packages/netmuxd/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jkcoxson/netmuxd
+TERMUX_PKG_DESCRIPTION="A replacement/addition to usbmuxd which is a reimplementation of Apple's usbmuxd on MacOS"
```

Comment:
I can't find any package (or package request) for this package.
https://repology.org/projects/?search=netmuxd&maintainer=&category=&inrepo=&notinrepo=&repos=&families=&repos_newest=&families_newest=

This may be a better fit for the [TUR](https://github.com/termux-user-repository/tur).